src/Nethermind/Nethermind.Db/ITunableDb.cs

```diff
@@ -9,12 +11,19 @@ public interface ITunableDb
 
     enum TuneType
```

Comment:
@asdacap I need help writing docs for these options. I checked out the comments in [DbOnTheRocks.cs#L1492-L1540](https://github.com/NethermindEth/nethermind/blob/feature/docs-enums/src/Nethermind/Nethermind.Db.Rocks/DbOnTheRocks.cs#L1492-L1540) but I'm not sure I understand them clearly. We need something short and end-user-friendly; otherwise, hide it from the docs.

Reply:
Yea, this probably should be hidden.

Reply:
I hid the `Sync.TuneDbMode` and `Sync.BlocksDbTuneDbMode` options from docs.